auction9-backend/handler.js

```diff
@@ -87,6 +87,25 @@ export const postAuction = async (event, context) => {
   }
 };
 
+/* getAuctionBids - returns auction bids (users' bids history)
+ * GET: auctions/{id}/bids
+ */
+export const getAuctionBids = async (event, context) => {
+  try {
+    // once we enable bidding, remove hardcoded auctionID
+    // let resultAuctionBids = await mysql.query('SELECT u.name, price, time FROM tbl_user_auction ua JOIN tbl_user u on (ua.userID = u.userID) WHERE auctionID=? ORDER BY price', [auctionID]);
+    let resultAuctionBids = await mysql.query('SELECT u.name, price, time FROM tbl_user_auction ua JOIN tbl_user u on (ua.userID = u.userID) WHERE auctionID=1 ORDER BY price');
```

Comment:
Are we sure that we want to use userId in this query and to do join per user. I think that this query should return all bids no matter on user id, only actionId is required.